recipes/pytim/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pytim" %}
+{% set version = "0.5.5" %}
+{% set sha256 = "6c9b4cd137d876b811430b80109f969ead85567bcf2a383be21982cddb606f44" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 1
+  skip: True  # [py3k or win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - toolchain
+  run:
```

Comment:
The `setup.py` has version restrictions that should probably be included here:

https://github.com/Marcello-Sega/pytim/blob/master/setup.py#L93-L95

Reply:
first, thanks a lot for the prompt feedback!

I did include them initially, but travis complained saying

Error: bad character '>' in package name dependency 'scipy>0.18'

https://travis-ci.org/conda-forge/staged-recipes/builds/287534953#L200

I will try again adding spaces

Reply:
You need a space between the version and the package